CHANGELOG.md

```diff
@@ -8,9 +8,15 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 ## Unreleased
 
 ### Added
```

Comment:
I was wondering if we should provide also `axi_lite_cut` and `axi_lite_multicut` modules to complement the AXI4-Lite story. What do you think?

Reply:
That’s one of the advantages of using structs for the interfaces: We do not need a separate `_lite_cut` as the exakt type of the bus does not matter, as long as it provides `valid/ready` 😊

Reply:
Completely forgot about that. That's great!